src/main/webapp/app/module/primary/landscape/Landscape.component.ts

```diff
@@ -592,6 +598,44 @@ export default defineComponent({
       return document?.activeElement?.tagName === 'INPUT';
     };
 
+    const performSearch = (query: string) => {
+      highlightModule(query);
+    };
+
+    const highlightModule = (query: string) => {
+      if (!query) {
+        highlightedModule.value = null;
+        nextTick().then(resetLandscapeContainerPosition);
+        return;
+      }
+
+      const foundModule = findModule(query);
+      highlightedModule.value = foundModule ? new ModuleSlug(foundModule) : null;
+
+      if (foundModule) {
+        const moduleElement = landscapeElements.value.get(foundModule)!;
+        nextTick().then(() => scrollToHighlightedModule(moduleElement));
+      }
+    };
+
+    const findModule = (query: string): string | null => {
+      return [...landscapeElements.value.keys()].find(key => key.toLowerCase().includes(query.toLowerCase())) || null;
```

Comment:
```suggestion
      return [...landscapeElements.value.keys()].find(key => key.toLowerCase().includes(query.toLowerCase())) ?? null;
```
![image](https://github.com/user-attachments/assets/65a06b1c-9afb-41c2-bbf4-c6bd2811ae6d)